jenkins-pipelines/features-full-cluster-replacement.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    availability_zone: 'a,b',
+    backend: 'aws',
+    region: '''["eu-west-1", "eu-west-2"]''',
+    test_name: 'full_cluster_replacement_test.FullClusterReplacement.test_rolling_full_cluster_replacement_add_then_remove_nodes',
```

Comment:
In all relevant places, it needs to be clear that replacement here is by adding and decommissioning.
(Which isn't really a node replacement...)

Reply:
Done